### DISCUSSION TITLE
Is there a way to parse the JSON to HTML?

### DISCUSSION THREAD

**Discussion by adamhinckley** [react-page/react-page#1127] [Q&A]
I am looking at using this library in a case where a user creates custom content in a WYSISYG editor where the editing happens in app 1 and the customizations are rendered in app 2.

Is there a way to parse the JSON this library outputs in app 1 into HTML for app 2?

**Comment by zehawki** [comment]
Not directly. As we've mentioned here https://react-page.github.io/docs/#/editor?id=internal-json-details the internal JSON is not meant for parsing by a 3rd party tool. It does not actually store information that can be easily converted back except with react-page itself. For example - some of the info actually tells _how_ to render something at run time, _which_ existing React block to use and so on. If those precise blocks are not available, nothing can be rendered.

Having said that, one could have a go at creating a parser for the simplest components eg text and images.

But overall, your simplest method is to have app 2 also use react-page.
